src/sugar3/graphics/objectchooser.py

```diff
@@ -99,22 +102,20 @@ def get_preview_pixbuf(preview_data, width=-1, height=-1):
 
 
 class ObjectChooser(object):
+    """
+    UI interface for objec choosers.
 
-    def __init__(self, parent=None, what_filter=None, filter_type=None,
-                 show_preview=False):
-        """Initialise the ObjectChoser
-
-        parent -- the widget calling ObjectChooser
+    Object choosers can be used by acivities to allow the user to select objects
+    from the file system or from some other similar source.
 
-        what_filter -- string
+    Keyword Args:
+        parent (:class:`Gtk.Widget`): the widget calling ObjectChooser
```

Comment:
What should this be usually?  Should it be the sugar3.activity.activity.Activity (self for some activities)?  Maybe give a usage example?